convert_hf_to_gguf.py

```
@@ -5960,6 +5963,68 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iter
+
+        # SAM configuration
+        sam_hparams = hparams['sam']
+        self.gguf_writer.add_vision_sam_layers_count(sam_hparams['layers'])
+        self.gguf_writer.add_vision_sam_embedding_length(sam_hparams['width'])
+        self.gguf_writer.add_vision_sam_head_count(sam_hparams['heads'])
+
+    def get_vision_config(self) -> dict[str, Any]:
+        vision_config: dict[str, Any] | None = self.global_config.get("vision_config")
+
+        if not vision_config:
+            raise ValueError("DeepseekOCR model requires 'vision_config' in the model configuration, but it was not found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)